app/admin/tenants/onboarding.py

```diff
@@ -0,0 +1,203 @@
+from typing import Optional, List
+from aries_cloudcontroller import (
+    AcaPyClient,
+    InvitationCreateRequest,
+)
+from aries_cloudcontroller.model.create_wallet_token_request import (
+    CreateWalletTokenRequest,
+)
+from fastapi.exceptions import HTTPException
+from pydantic import BaseModel
+from pydantic.networks import AnyHttpUrl
+from app.admin.tenants.models import UpdateTenantRequest
+from app.dependencies import Role, get_tenant_controller, get_yoma_controller
+from app.facades.trust_registry import (
+    TrustRegistryRole,
+    actor_by_id,
+    update_actor,
+)
+
+from app.error import CloudApiException
+from app.facades import acapy_ledger, acapy_wallet
+from app.util.did import qualified_did_sov
+
+
+class OnboardResult(BaseModel):
+    did: str
+    didcomm_invitation: Optional[AnyHttpUrl]
+
+
+async def handle_ecosystem_tenant_update(
+    admin_controller: AcaPyClient,
+    tenant_id: str,
```

Comment:
The `tenant_id` is not part of the API json body, because it is a path param instead of a body param. If we add it to the `UpdateTenantRequest` model it will also be part of the API call body, which would be confusing (it's in both the path and the body). And according to restful principles the id should be part of the path (as it describes the resource we would like to update)

Reply:
Ok makes sense. Thanks for clarifying